components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -32,7 +32,7 @@ function TeamUsage() {
     const team = getCurrentTeam(location, teams);
     const [billedUsage, setBilledUsage] = useState<BillableSession[]>([]);
     const [currentPage, setCurrentPage] = useState(1);
-    const [resultsPerPage] = useState(10);
+    const [resultsPerPage] = useState(1);
```

Comment:
I removed the dummy file and instead set this to 1 per page. Will change to `15` before merging.

Reply:
Already put back and will unhold.